.github/workflows/ci-cd.yml

```diff
@@ -177,8 +177,9 @@ jobs:
         python-version: ${{ matrix.pyver }}
     - name: Get pip cache dir
       id: pip-cache
+      shell: bash
```

Comment:
*(optional, non-blocking)* I like putting the `shell:` declarations after `run:` to keep the mapping keys sorted

Reply:
reordered